ultimatejobweb/migrations/test_data_jobs1.py

```diff
@@ -0,0 +1,39 @@
+from django.db import migrations, transaction
+from ultimatejobweb.models import Job, Company
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ultimatejobweb', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        company1 = Company(1, '"https://careers-redhat.icims.com/jobs/search?ss=1&in_iframe=1&searchKeyword=',
+                           'logo1', 'function1')
+        company2 = Company(2, 'https://www.amazon.jobs/en/', 'logo2', 'function2')
```

Comment:
you need to save the data for both of these, see .save() at the end of this
```
company1 = Company(1, '"https://careers-redhat.icims.com/jobs/search?ss=1&in_iframe=1&searchKeyword=',
                           'logo1', 'function1').save()
```

Reply:
You also don't have the company names here. 
You're also creating more company records below. 
where do you want to create the company records?